pegasus/sites.v3/hourofcode.com/views/survey_prize_choice.haml

```diff
@@ -0,0 +1,18 @@
+- if value == 'None' || prize_available?(value)
+
+  / everything before the '.', i.e. 'Amazon.US' -> 'Amazon'
+  - base_value = value.partition('.').first
+  %label{class: "radio-inline"}
+    %input{type: "radio", id: "prize-choice-#{base_value.downcase}", name: "prize_choice_s", value: value}
+      - case base_value
+        - when 'Microsoft'
+          $10.00 Windows Store credit
+        - when 'Apple'
+          $10.00 iTunes credit
+        - when 'Amazon'
+          $10.00 Amazon.com credit
```

Comment:
according to spec. I double-checked with Laura. That's how it was marketed on hoc.com. Good suggestion for next time.